client/src/pages/GitHubStats.tsx

```
@@ -0,0 +1,35 @@
+          console.error("Failed to fetch GitHub stats:", error);
+        }
+      };
+
+      fetchGithubStats();
+    }
+  }, [githubUsername]);
+
+  return (
+    <div>
+      <h2 className="text-xl font-bold">GitHub Streak Stats</h2>
+      {streakStat ? <div dangerouslySetInnerHTML={{ __html: streakStat }} /> : <Skeleton className="h-48 w-full" />}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)